raiden-ts/src/transfers/epics.ts

```diff
@@ -1025,36 +1181,48 @@ export const transferReceivedReplyProcessedEpic = (
 };
 
 /**
- * Receiving RefundTransfer for pending transfer fails it
+ * When receiving a WithdrawRequest message, create the respective withdrawReceiveRequest action
```

Comment:
`[[WithdrawRequest]]` although not sure if our typedoc process links this correctly